### DISCUSSION TITLE
Second battery

### DISCUSSION THREAD

**Discussion by Errator** [snitzelweck92/homeassistant_alphaess_modbus_tcp#2] [Q&A]
Hello and many thanks for your work,

i have a second battery. Is your hint that for that you have it tested with only 1 battery or does it mean it will not work with a second one.

Do i have to edit something for the second battery?

Many thanks.

Mario

**Comment by snitzelweck92** [comment]
Hi :)
I have only one battery in my system, so i can not say if it is working, but i think this should be nonproblem.
Because there are no additional register to read in the modbus for a second battery. For the BMS the batteries are "one big"  battery.
So you can try it and can make a remark here :) 

Thanks
